<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureFrameCreator</class>
 <widget class="QDialog" name="FeatureFrameCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>238</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>235</width>
    <height>238</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Feature Frame</string>
  </property>
  <widget class="QGraphicsView" name="Preview">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>29</y>
     <width>170</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ChoicesBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PickedTypeLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
